arisia-remote/app/arisia/fun/DuckService.scala

```diff
@@ -99,7 +99,7 @@ class DuckServiceImpl(
           )
         } else {
           // Somehow got a request from the wrong place, which seems hinky:
-          val error = s"Got a request for Duck $duckId from incorrect location $from!"
+          val error = s"Got a request for Duck $duckId from incorrect location $from! It's supposed to be ${duck.requestingUrl}"
```

Comment:
Should we not include the correct URL? If someone does guess the duck, but isn't at the right place, won't their network tab tell them where to go?

Reply:
oooh good call, i had put that in there for debugging but i don't want that